assets/themes/halloy.yaml

```diff
@@ -0,0 +1,11 @@
+name: "Halloy"
```

Comment:
Do we need `name` or can the filename just be the key?

Reply:
Name of the theme as the value is more user friendly in my opinion

Reply:
I think the way to go is:
- filename -> config file theme key
- `name` -> theme name to display in the app

Let's say we add the file `solarized_dark.yaml` for the `Solarized (Dark)` theme. `solarized_dark` is used as the key and `Solarized (Dark)` can be used to display in the future command bar of the app, e.g..

Reply:
In this PR `name` is not used anywhere, it's just there so users are already educated to add that instead of having to change their files in the future.

Reply:
Agreed that makes sense

Reply:
Should wiki be updated with `name` in the example themes?

Reply:
I think it should yeah, @jhff.